src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -412,6 +412,8 @@ public static class Importer
          public static readonly string MappingTitle = "Mapping";
          public static readonly string ReloadWillCauseChangeOfDataSets =
             "Reloading will cause the following changes in observed data. Do you really want to reload?";
+         public static readonly string UnexpectedExceptionWhenLoading =
+            "Unexpected exception when trying to load. File format probably not supported";
```

Comment:
I would not use the word "Exception" for user-facing errors

"An unexpected error occurred while loading the file. The file format is probably not supported. Please check the documentation for more details" 

And can we add the link to the documentation here?

Reply:
fixed :)